packages/phenomic-docs/App.js

```diff
@@ -0,0 +1,42 @@
+import React from "react"
+import { AppRegistry } from "react-native-web"
+import { Router, Route, browserHistory } from "react-router"
+
+import Header from "./components/Header"
+import Home from "./components/Home"
+import DocPage from "./components/DocPage"
+import APIPage from "./components/APIPage"
+import APIListPage from "./components/APIListPage"
+import ChangelogPage from "./components/ChangelogPage"
+import ChangelogListPage from "./components/ChangelogListPage"
+import ShowcaseList from "./components/ShowcaseList"
+import APITagListPage from "./components/APITagListPage"
+
+import { Renderer } from "phenomic-react/lib/client"
+
+const app = Renderer.createApp(
+  <Router history={ browserHistory }>
+    <Route component={ (props) => <div><Header />{props.children}</div> }>
+      <Route path="/" component={ Home } />
+      <Route component={ props => <div>{props.children}</div> }>
```

Comment:
@bloodyowl can you confirm this wrapper is useless here and for /changelog routes below?